install_config/configuring_sdn.adoc

```diff
@@ -178,3 +178,22 @@ so that the master does not schedule containers on it.
 Both options are presented as part of a practical use-case in the documentation
 for configuring xref:../install_config/routing_from_edge_lb.adoc#install-config-routing-from-edge-lb[routing from an
 edge load-balancer to containers within {product-title} SDN].
+
+[[using-flannel]]
+== Using Flannel
+As an alternative to the default SDN, {product-title} also provides Ansible
+playbooks for installing *flannel*-based networking. This is useful if running
+{product-title} within a cloud provider platform, such as OpenStack, and you
+want to avoid using dual Open vSwtich SDN on both platforms.
+
+To enable *flannel* within your {product-title} cluster, set the following
+variables in your Ansible inventory file before running the installation.
+
+----
+openshift_use_openshift_sdn=false
+openshift_use_flannel=true
+----
```

Comment:
Its not clear if this is or can be liked to [https://docs.openshift.com/container-platform/3.3/install_config/configuring_sdn.html#configuring-sdn-config-pod-network-ansible](https://docs.openshift.com/container-platform/3.3/install_config/configuring_sdn.html#configuring-sdn-config-pod-network-ansible)